packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -28,3 +28,4 @@ export const DEFAULT_DEV_HUB_USERNAME_KEY = 'defaultdevhubusername';
 export const CONFLICT_DETECTION_ENABLED = 'detectConflictsAtSync';
 export const BETA_DEPLOY_RETRIEVE = 'experimental.deployRetrieve';
 export const PKG_ID_PREFIX = '04t';
+export const USE_APEX_LIBRARY = 'experimental.useApexLibrary';
```

Comment:
Place config values next to each other? Ideally i'd like to see them grouped under an object but that's too much refactoring for now

Reply:
True I think it's rather confusing to have the config values spread out across multiple files, so it could be worth looking into refactoring. For now, I think it's best to keep this value within the `constants` file so we're at least following the current pattern to minimize more confusion

Reply:
Yeah, I mean in this file group the values together and maybe add a comment :)

The underlying idea is that these values are implicitly typed (getConfigValue would expect a config value rather than `string`)